index.html

```diff
@@ -19,23 +19,170 @@
+          <li>Ruby</li>
         </ul>
-    </section>
+      </div>
+      <button type="button" class="btn">See Project</button>
+    </article>
+
+  <!-- CARDS -->
+<section class="grid-container">
+  <section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section>
+
+  <section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
+  </section><section class="card-section">
+    <h2 class="Profesional">Profesional Art Printing Data</h2>
+    <p class="paragraph">
+      A daily selection of privately personalized reads; no accounts or
+      sign-ups required. has been the industry's standard
+    </p>
+    <ul class="tech-list">
+      <li>Html</li>
+      <li>Bootstrap</li>
+      <li>Ruby</li>
+    </ul>
+    <button type="button" class="btn1">See Project</button>
   </section>
+</section>
+</section>
+<section class="about-me">
+  <h1 class="logo-message">About me</h1>
```

Comment:
- [ ] Kindly change this h1 tag to any hx tag from h2- h6, it is better for your page to have only one h1 tag as the main heading of the page, this will make your project more semantic and easily readable by screen readers 👍🏿